formats/utils/color-factory/variants/back-office-variants.js

```diff
@@ -0,0 +1,739 @@
+const {saturationAdjustmentFn, hueRotationFn} = require('../color-factory');
+
+const backOfficeVariants = {
```

Comment:
Now not sure if I want to call this "back office" or something like base or default. Then that signals to other apps/platforms which may need add/override colors they can use an inheritance model. Otherwise an entirely separate config can be created.